tst/com/vivian/java/leetcode/Q1_TwoSumTest.java

```diff
@@ -0,0 +1,18 @@
+package com.vivian.java.leetcode;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit test for Q28_strStr. In eclipse, you can right click this file, and run
+ * as JUnit Test.
+ */
+public class Q1_TwoSumTest {
+    private Q1_TwoSum solution = new Q1_TwoSum();
+
+    @Test
+    public void test() {
```

Comment:
Please add more test cases. For example:
- {2, 7, 11, 15}, 20. What should be the output?
- {2, 7, 11, 15}, 14. What should be the output?